.flowconfig

```diff
@@ -0,0 +1,9 @@
+[ignore]
```

Comment:
Yea, when we run `flow-bin` manually we might be able to avoid this. Not sure tho!